modules/updating-control-plane-only-update.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/control-plane-only-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-control-plane-only-update_{context}"]
+= Control Plane Only update
+
+The following procedure pauses all non-master machine config pools and performs updates from {product-title} <4.y> to <4.y+1> to <4.y+2>, then unpauses the previously paused machine config pools.
```

Comment:
IMO you could probably trim out this phrase for less awkwardness, especially since this process is described in detail later in the procedures:
```suggestion
The following procedure pauses all non-master machine config pools and performs updates from {product-title} <4.y> to <4.y+1> to <4.y+2>, then unpauses the machine config pools.
```